metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/configuration/MetadataImporterMainApplicationConfiguration.java

```diff
@@ -43,25 +44,25 @@
 
     @Bean
     @ConditionalOnProperty(name = "import.source", havingValue = "API")
-    public ObjectsImporter objectImporterForNonEga(SraXmlRetrieverByAccession sraXmlRetrieverByAccession,
+    public ObjectsImporter objectImporterForNonEga(SraXmlRetrieverThroughApi sraXmlRetrieverThroughApi,
```

Comment:
These method names are not descriptive of their functionality, what about renaming them as `objectImporterFromEnaApi` and `objectImporterFromEnaDatabase`?